Index: app/src/main/java/com/example/sweetbakes/Home/DashboardUI.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/example/sweetbakes/Home/DashboardUI.kt b/app/src/main/java/com/example/sweetbakes/Home/DashboardUI.kt
--- a/app/src/main/java/com/example/sweetbakes/Home/DashboardUI.kt	
+++ b/app/src/main/java/com/example/sweetbakes/Home/DashboardUI.kt	(date 1742716655756)
@@ -28,7 +28,6 @@
 import androidx.compose.ui.text.style.TextAlign
 import androidx.compose.ui.text.style.TextDecoration
 import androidx.compose.ui.text.withStyle
-import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
 import androidx.lifecycle.viewmodel.compose.viewModel
@@ -112,7 +111,7 @@
                         modifier = Modifier
                             .size(140.dp)
                             .graphicsLayer(scaleX = -1f),
-                        colorFilter = androidx.compose.ui.graphics.ColorFilter.tint(MaterialTheme.colorScheme.onSurface)
+                        colorFilter = ColorFilter.tint(MaterialTheme.colorScheme.onSurface)
                     )
 
                     Spacer(modifier = Modifier.width(8.dp))
@@ -233,7 +232,11 @@
 }
 
 @Composable
-fun TaskCard(title: String, count: Int, color: Color) {
+fun TaskCard(
+    title: String,
+    count: Int,
+    color: Color
+) {
     Card(
         modifier = Modifier
             .width(140.dp)
